tg.js - Update 5.7.0

Navsy17 opened the pull request.

Commit:
tg.js - Update 5.7.0

callmedandann commented:
Line 567: Bumalik sa huling checkpoint
Line 569: Lumipad
Line 571: Patayin
Line 685: Mga papremyo 
Line 699: Walang nahanap na laro<br><small>I-try na ibahin ang Quickplay options</small>

callmedandann commented:
Line 567: Bumalik sa huling checkpoint
Line 569: Lumipad
Line 571: Patayin
Line 685: Mga papremyo
Line 699: Walang nahanap na laro
I-try na ibahin ang Quickplay options

Jaehnvnct commented:
Line 754: Itago ang mga username (Allies)
Line 883: Hindi makapag-host ng laro habang nasa Tayaan
Line 1009: Malapit lang ang gintong itlog
Line 1010: Nawala na ang gintong itlog
Line 1012: Nakita mona ang itlog na ito!
Line 1013: Natapos mo ang paghahanap ng itlog!

Commit:
Dandan #1 & Jaehn #1

Dandan - 5 lines
Jaehn - 1 line (other suggestions have been committed already)

Commit:
@KebbsInGit's changes

the gist of Kebb's changes from #483. Around 23-24 lines.

Navsy17 commented:
Navsy  - 5
Dandan - 5
Jaehn - 1 (+4 for still contributing Golden egg changes)
Kebbs - 24 

total (July 11): 39 lines have been changed

Markzt commented:
line 57 : ilagay ang password
line 684 : upang makumpleto ang hamon

Markzt commented:
lines 683 : mga hamon

Commit:
Markzt (3 lines)

Navsy17 commented:
still ongoing, been busy with other stuff sorry for slow updates

Commit:
Navsy Changes

Navsy17 commented:
Navsy - 58
Dandan - 5
Jaehn - 1 (+4 for still contributing Golden egg changes)
Kebbs - 24
Markzt - 3

StarRayX commented on tg.js:
```diff
@@ -77,10 +77,10 @@ purchase.error May naganap na error
 settings.reset Mag-reset
-settings.reset.confirm Sigurado ka bang mag-reset ng inyong settings?
+settings.reset.confirm Sigurado ka ba na i-reset ang inyong settings?
 settings.import iangkat
```
```suggestion
settings.import i-import
```

StarRayX commented on tg.js:
```diff
@@ -77,10 +77,10 @@ purchase.error May naganap na error
-settings.reset.confirm Sigurado ka bang mag-reset ng inyong settings?
+settings.reset.confirm Sigurado ka ba na i-reset ang inyong settings?
 settings.import iangkat
 settings.export iluwas
```
```suggestion
settings.export iexport
```

Not used in proper context, pag-angkat and pag-luwas is for trade/economic contexts

StarRayX commented on tg.js:
```diff
@@ -279,7 +279,7 @@ settings.crosshairm.color Kulay
 settings.crosshairm.shadow.thickness Shadow Thickness
 settings.crosshairm.size Laki
-settings.crosshairm.thickness Thickness
+settings.crosshairm.thickness Gaano Kakapal
```
```suggestion
settings.crosshairm.thickness Kakapalan
```

StarRayX commented on tg.js:
```diff
@@ -830,8 +830,8 @@ social.market.gift Magregalo ng KR
 social.market.gift.msg Mag-iwan ng mensahe
 respawn.auto Auto Respawning sa {0}
```
```suggestion
respawn.auto Kusang Magrerespawn sa {0}
```

StarRayX commented on tg.js:
```diff
@@ -830,8 +830,8 @@ social.market.gift Magregalo ng KR
-funds.insufficient Hindi sapat na KR
+custom.host.wager Hindi pwede mag-host habang mayroong wager
+funds.insufficient Hindi sapat ang KR
 funds.out Ubos ang KR
```
```suggestion
funds.out Ubos na ang iyong KR.
```

"It appears you have run out of Krunkies." was the original, it included second-person perspective

StarRayX commented on tg.js:
```diff
@@ -130,7 +130,7 @@ settings.chat.chatBGOp Chat BG Opacity
 settings.chat.chatHeight Tangkad ng Chat
 settings.chat.chatTimer Orasan ng Pag-chat
```
```suggestion
settings.chat.chatTimer Pagitan ng Pag-chat
```

Navsy17 replied:
actually a good alternative than going literal, thanks alot

StarRayX commented on tg.js:
```diff
@@ -130,7 +130,7 @@ settings.chat.chatBGOp Chat BG Opacity
 settings.chat.chatHeight Tangkad ng Chat
 settings.chat.chatTimer Orasan ng Pag-chat
 settings.chat.chatTextOutline Chat Text Outline
```
```suggestion
settings.chat.chatTextOutline Outline ng Chat Text
```

StarRayX commented on tg.js:
```diff
@@ -77,10 +77,10 @@ purchase.error May naganap na error
 settings.import iangkat
 settings.export iluwas
-settings.none Walang Nahanap na Mga settings
+settings.none Hindi matagpuan ang settings
```
```suggestion
settings.none Hindi mahanap ang settings
```

StarRayX commented on tg.js:
```diff
@@ -279,7 +279,7 @@ settings.crosshairm.color Kulay
 settings.crosshairm.size Laki
-settings.crosshairm.thickness Thickness
+settings.crosshairm.thickness Gaano Kakapal
 settings.crosshairm.gap Gap
```
```suggestion
settings.crosshairm.gap Pagitan
```

StarRayX commented on tg.js:
```diff
@@ -308,12 +308,12 @@ settings.gameprivacy.header Pagkapribado ng Laro
 settings.gameprivacy.streamer Mode ng Streamer
```
```suggestion
settings.gameprivacy.streamer Streamer Mode
```
for uniformity with anonymous mode

StarRayX commented on tg.js:
```diff
@@ -308,12 +308,12 @@ settings.gameprivacy.header Pagkapribado ng Laro
 settings.gameprivacy.partner Ipakita: Partnered Badge
 settings.gameprivacy.rank Ipakita: Ranked Badge
-settings.gameprivacy.custom Ipakita: Custom Badge
+settings.gameprivacy.custom Ipakita: Kustom Badge
```
```suggestion
settings.gameprivacy.custom Ipakita: Custom Badge
```
please revert, there's no need to change the c in custom on this line when there are 100 other lines with 'Custom' and not 'Kustom'

StarRayX commented on tg.js:
```diff
@@ -349,8 +349,8 @@ settings.audio.sound Bolyum o Tunog Pangkalahatan
-settings.audio.micquality Mic Quality
+settings.audio.mic Bolyum ng Mic
+settings.audio.micquality Kalidad ng Mic
 settings.audio.weps Bolyum ng Tunog ng Armas
```
```suggestion
settings.audio.weps Bolyum ng mga Armas
```

StarRayX commented on tg.js:
```diff
@@ -680,9 +680,9 @@ windows.client.linux Installer sa Linux
+windows.chall.header Mga Hamon
+windows.chall.login maglog-in upang makumpleto ang mga hamon!
+windows.spin.header Mga Papremyo
 thumbnail.upload.limit Failed. {0}kb Thumbnail Limit
```
```suggestion
thumbnail.upload.limit Failed. {0}kb Limitasyon sa mga Thumbnail```

StarRayX commented on tg.js:
```diff
@@ -680,9 +680,9 @@ windows.client.linux Installer sa Linux
+windows.chall.login maglog-in upang makumpleto ang mga hamon!
+windows.spin.header Mga Papremyo
 thumbnail.upload.limit Failed. {0}kb Thumbnail Limit
 image.upload.limit Failed. {0}kb ang Limitasyon ng Larawan
```
```suggestion
image.upload.limit Failed. {0}kb ang Limitasyon sa mga Larawan
```

StarRayX commented on tg.js:
```diff
@@ -738,7 +738,7 @@ server.config.t3Dmg Damage ng Pangkat 3
 server.config.t4Dmg Damage ng Pangkat 4
 server.config.t5Dmg Damage ng Pangkat 5
 server.config.selTeam Piliin Ang Pangkat 
```
```suggestion
server.config.selTeam Pumili ng Pangkat 
```

StarRayX commented on tg.js:
```diff
@@ -738,7 +738,7 @@ server.config.t3Dmg Damage ng Pangkat 3
+server.config.frFire Pwedeng Barilin ang Kakampi
 server.config.allowSpect Panonood ng Laro
 server.config.killRewards Gantimpala kapag nakapatay
 server.config.headshotOnly Mga headshot Lamang
```
```suggestion
server.config.headshotOnly Mga Headshot Lamang
```

StarRayX commented on tg.js:
```diff
@@ -738,7 +738,7 @@ server.config.t3Dmg Damage ng Pangkat 3
-server.config.frFire Friendly Fire
+server.config.frFire Pwedeng Barilin ang Kakampi
 server.config.allowSpect Panonood ng Laro
 server.config.killRewards Gantimpala kapag nakapatay
```
```suggestion
server.config.killRewards Gantimpala kapag may napatay
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
+commands.forfeit.next Kailangan maglaro muna ng isang beses
 commands.forfeit.wait Hintayin munang magsimula ang laro upang maka-forfeit 
 commands.forfeit.later Ulitin nalang muli mamaya
 commands.forfeit.voted Ikaw ay bumoto na.
```
```suggestion
commands.forfeit.voted Nakaboto ka na.
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
 commands.forfeit.voted Ikaw ay bumoto na.
-commands.forfeit.limit Isang beses kalang makababoto sa isang laro
+commands.forfeit.limit Isang beses kalang makaboboto sa isang laro
 commands.forfeit.ended Iniwan na ang laro
```
```suggestion
commands.forfeit.ended Sumuko na sa laro
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
-commands.forfeit.limit Isang beses kalang makababoto sa isang laro
+commands.forfeit.limit Isang beses kalang makaboboto sa isang laro
 commands.forfeit.ended Iniwan na ang laro
 commands.forfeit.ended.other Umalis na ang kabilang grupo
```
```suggestion
commands.forfeit.ended.other Sumuko na ang kalaban
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
+commands.forfeit.limit Isang beses kalang makaboboto sa isang laro
 commands.forfeit.ended Iniwan na ang laro
 commands.forfeit.ended.other Umalis na ang kabilang grupo
 server.message.join si {0} ay sumali sa laro
```
```suggestion
server.message.join Si {0} ay sumali sa laro
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
 commands.forfeit.ended Iniwan na ang laro
 commands.forfeit.ended.other Umalis na ang kabilang grupo
 server.message.join si {0} ay sumali sa laro
 server.message.left si {0} ay umalis sa laro
```
```suggestion
server.message.left Si {0} ay umalis sa laro
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
 commands.forfeit.ended.other Umalis na ang kabilang grupo
 server.message.join si {0} ay sumali sa laro
 server.message.left si {0} ay umalis sa laro
 server.message.kicked si {0} ay natanggal sa laro
```
```suggestion
server.message.kicked Si {0} ay natanggal sa laro
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
 server.message.join si {0} ay sumali sa laro
 server.message.left si {0} ay umalis sa laro
 server.message.kicked si {0} ay natanggal sa laro
 server.message.banned si {0} ay na-ban
```
```suggestion
server.message.banned Si {0} ay na-ban
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
 server.message.left si {0} ay umalis sa laro
 server.message.kicked si {0} ay natanggal sa laro
 server.message.banned si {0} ay na-ban
 server.message.votekicked na vote-kick si {0} 
```
```suggestion
server.message.votekicked Na-vote kick si {0} 
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
-event.easter.found.new {0} found {1}/{2} Eggs
-event.easter.found.old You already discovered this egg!
-event.easter.completed {0} completed the Egg hunt!
+server.message.arrested si {0} ay naaresto ng K.P.D
```
```suggestion
server.message.arrested Si {0} ay naaresto ng K.P.D
```

StarRayX commented on tg.js:
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
+server.message.arrested si {0} ay naaresto ng K.P.D
+event.easter.golden.spawn Nagkaroon ng Gintong Itlog
+event.easter.golden.despawn Nawala ang Gintong Itlog
+event.easter.found.new Si {0} aynakahanap ng {1}/{2} na Gintong Itlog
```
```suggestion
event.easter.found.new Nakahanap si {0} ng {1}/{2} na Gintong Itlog
```

Commit:
27 lines from Goos (welcome back!)

Goos' changes

Co-authored-by: StarRayX <40836712+StarRayX@users.noreply.github.com>

Navsy17 commented:
Navsy - 57
Dandan - 5
Jaehn - 1 (+4 for still contributing Golden egg changes)
Kebbs - 24
Markzt - 3
Goos/StarRayX - 27

Commit:
Update tg.js

Navsy17 commented:
Navsy - 70
Dandan - 5
Jaehn - 4
Kebbs - 24
Markzt - 3
Goos/StarRayX - 27

mty22 commented:
@kebbs please reach out to me with your ign.

mty22 commented:
your total came to 102% lol. it's fine though.
